src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  
+actions.forEach((action) => {
+    switch (action.type) {
+      case 'addProperties':
+        const extraData = action.extraData;
+        Object.keys(extraData).forEach((key) => {
+          state[key] = extraData[key];
+        });
+        break;
+      case 'removeProperties':
+        const keysToRemove = action.keysToRemove;
```

Comment:
You can use destructuring 
```suggestion
        const { keysToRemove } = action;
```